base/system/userinit/lang/de-DE.rc

```diff
@@ -1,3 +1,11 @@
+/*
+ * PROJECT:     ReactOS Userinit Logon Application
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     German resource file
+ * TRANSLATORS: Copyright 2005 Aleksey Bragin <aleksey@reactos.org>
```

Comment:
I doubt that Aleksey did German translations in the past.

Reply:
He committed translation from Kevin Schnabel. I will fix it.